<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
